source/docs/contributing/frc-docs/contribution-guidelines.rst

```diff
@@ -20,8 +20,32 @@ These docs serve to provide a learning ground for all *FIRST* Robotics Competiti
 
 Please see the :ref:`docs/contributing/frc-docs/style-guide:Style Guide` for information on styling your documentation.
 
-Note on New Articles
+Creating New Content
 --------------------
-All new articles will undergo a review process before being merged into the repository. This review process will be done by members of the WPILib team. New Articles must be on official *FIRST* supported Software and Hardware. Documentation on unofficial libraries or sensors *will not* be accepted. This process may take some time to review, please be patient.
 
-Minor revisions and rewrites will be accepted at a more generous rate.
+Thanks for contributing to the `frc-docs <https://github.com/wpilibsuite/frc-docs>`__ project! There are a couple things you should know before getting started!
+
+Where to place articles?
+^^^^^^^^^^^^^^^^^^^^^^^^
+
+The location for new articles can be a pretty opinionated subject. Standalone articles that fall well into an already subject category should be placed into mentioned subject category (documentation on something about simulation should be placed into the simulation section). However, things can get pretty complicated when an article combines or references two separate existing sections. In this situation, we advise the author to open an issue on the repository to get discussion going before opening the PR.
+
+.. note:: All new articles will undergo a review process before being merged into the repository. This review process will be done by members of the WPILib team. New Articles must be on official *FIRST* supported Software and Hardware. Documentation on unofficial libraries or sensors *will not* be accepted. This process may take some time to review, please be patient.
+
+Where to place sections?
```

Comment:
It wasn't immediately obvious to me that sections meant in the main TOC, rather then within a document. 
I'm also not sure this whole section couldn't be simplified to say open an issue for discussion for any major contribution before taking the trouble to make a PR.

Reply:
Well, that only applies for articles that don't already have a topic. I don't really think it's necessary to have "open an issue" as a mandatory prerequisite for contributing a new article.